flight/Libraries/math/pid.h

```diff
@@ -32,6 +32,13 @@
 #ifndef PID_H
 #define PID_H
 
+struct pid_deadband {
+	float dbW;				// Deadband width in degrees.
+	float dbB;				// Deadband slope in % (0..1).
```

Comment:
Would also prefer longer names so it's obvious when reading code that uses these members. Don't want to have to refer back to struct defs all the time to see what the members mean.